templates/partial/_filters.html

```diff
@@ -109,14 +109,6 @@ <h3 class="p-card__title p-heading--4 u-no-margin--bottom entity-card-title"></h
   </div>
 </template>
 
-<template id="no-results-message">
-  <div class="p-notification--information">
-    <p class="p-notification__response">
-      We couldn't find anything for your search<span class="search-query"></span>.
```

Comment:
Shouldn't we keep this message?

Reply:
It's never seen since the search results are now on their own page

Reply:
I thought about it as well but since we have now a different template for searches we don't need it.

Reply:
Ah of course :+1: